packages/react-ui-components/package.json

```diff
@@ -36,6 +36,7 @@
     "lodash.debounce": "^4.0.8",
     "lodash.isfunction": "^3.0.8",
     "lodash.omit": "^4.3.0",
+    "lodash.escaperegexp": "^4.1.0",
```

Comment:
Might make sense to use the same version as here: https://github.com/neos/neos-ui/blob/master/package.json#L87